bundler/spec/commands/update_spec.rb

```diff
@@ -527,6 +539,10 @@
       expect(the_bundle).to include_gems("activesupport 6.0.4.1", "tzinfo 1.2.9")
       expect(lockfile).to eq(expected_lockfile)
 
+      # needed because regressing to versions already present on the system
+      # won't add a checksum
+      expected_lockfile = expected_lockfile.gsub(/ sha256-[a-f0-9]+$/, "")
```

Comment:
Why? Is it because the original package may not be available? Could we store the original checksum of installed packages somewhere, so that we can resurrect that on this kind of downgrades to installed versions?

Does the same thing happens when upgrading to installed versions? I feel this kind of flip flop is going to be surprising?